deploy/docs/v1_migration_doc.md

```diff
@@ -0,0 +1,178 @@
+
+Some use-cases include : 
+ - custom log pipelines, 
+ - adding Fluentd filter plugins (ex: fluentd throttle plugin), or 
+ - adding Fluentd output plugins (ex: forward to both Sumo and S3)
+
+You can look for example configurations [here](../docs/v1_conf_examples.md)
+
+The Fluentd `deployments` have been changed to `statefulsets` to support the use of persistent volumes. This will allow better buffering behavior. They also now include `“fluentd”` in their names. This is not a breaking change for Helm users.
+
+The unified Fluentd `statefulsets` have been split into set of two different Fluentd's, one for `logs` and the other one for `metrics`.
+
+### How to upgrade
+**Note: The below steps are using Helm 2. Helm 3 is not supported.**
+#### 1. Upgrade to helm chart version `v0.17.3 `
+
+Run the below command to fetch the latest helm chart:
+```bash
+helm repo update
+
+If the user is not already on `v0.17.3` of the helm chart, upgrade to that version first by running the below command.
+
+```bash
+helm upgrade collection sumologic/sumologic --reuse-values --version=0.17.3
+```
+#### 2: Run upgrade script
+
+For Helm users, the only breaking changes are the renamed config parameters.
+For users who use a `values.yaml` file, we provide a script that users can run to convert their existing `values.yaml` file into one that is compatible with the major release. 
+
+
+- Get the existing values for the helm chart and store it as `current_values.yaml` with the below command:
+```bash
+helm get values <RELEASE-NAME> > current_values.yaml
+```
+- Run `curl` the upgrade script as follows:
+```bash
+curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v1.0/deploy/helm/sumologic/upgrade-1.0.0.sh
+```
+- Run the upgrade script on the above file with the below command.
+```bash
+./upgrade-1.0.0.sh current_values.yaml
+```
+- At this point, users can then run: 
+```bash
+helm upgrade collection sumologic/sumologic --version=1.0.0 -f new_values.yaml
+```
+## Non-Helm Users
+### Breaking Changes
+- The use of environment variables to set configs has been removed to avoid the extra layer of indirection and confusion. Instead, configs will be set directly within the Fluentd pipeline.
+- `kubernetesMeta` and `kubernetesMetaReduce` have been removed from `logs.kubernetes.sumologic.filter.conf` of the Fluentd pipeline for the same reason as above (Helm users)
+- Similarly `addStream` and `addTime` (default values were `true`) have been removed from `logs.kubernetes.sumologic.filter.conf` of the Fluentd pipeline; the default behavior will remain the same. To preserve the behavior of `addStream = false` or `addTime = false`, you can add: 
+```yaml
+<filter containers.**>
+  @type record_modifier
+  remove_keys stream,time
+</filter>
+```
+above the output plugin section [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/release-v1.0.0/deploy/kubernetes/fluentd-sumologic.yaml.tmpl#L251)
```

Comment:
This link doesn't work, the release branch name does not contain the patch number, so it should be `release-v1.0`